src/javascript/binary_japan/symbols.js

```diff
@@ -0,0 +1,20 @@
+if(typeof is_japan === 'function'){
+	Symbols._details = Symbols.details.bind({});
+
+	Object.defineProperties(Symbols,{
+		details:{
+			value:function(data){
+				var active_symbols = [];
+
+				data.active_symbols.forEach(function(symbol){
```

Comment:
Nice) I am familiar to it's principles, Perl has the same function. Now will use it in JS